src/Bridge/Symfony/Bundle/Resources/config/data_collector.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:twig="http://symfony.com/schema/dic/twig"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="ApiPlatform\Core\Bridge\Symfony\Bundle\DataCollector\RequestDataCollector" public="false">
```

Comment:
AFAIK, we don't use the id = class name convention yet in API Platform (we use aliases to enable autowiring instead, so the list of autowirable services isn't polluted). May I suggest `api_platform.data_collector.request` instead?